### PR TITLE
palm oil repair

### PR DESCRIPTION
**Pull request by aleene**
Repaired the palm oil entry for better detection.

### CODE REVIEW

**Comment by aleene** [comment]
I'll start on the vegan/vegetarian next.

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -20410,19 +20407,37 @@ it:oli vegetali non idrogenati
 fr:huile végétale hydrogénée
 de:pflanzliches Öl gehärtet
 
+<<<<<<< HEAD
```
Comment (hangy) [inline taxonomies/ingredients.txt]: Can you fix this merge conflict, please?

**Comment by stephanegigandet** [comment]
@aleene : for palm oil, we are going to use the from_palm:en:yes property instead of making en:palm a parent of everything.

There shouldn't be an ingredient "palm" in fact, we can keep it for now to make sure that we get all formulations of "some form of oil (palm)". So let's keep it under <en:palm oil

**Comment by aleene** [comment]
I moved **palm** as a parent of **palm oil**, as I did not want to make any implicit assumptions. **palm** might be **palm oil** (and probably is), but you never know. I wanted to take the conservative approach.

**Comment by teolemon** [comment]
``` 
#   Failed test at t/ingredients.t line 338.
#     Structures begin differing at:
#          $got->{ingredients_tags}[3] = 'en:palm'
#     $expected->{ingredients_tags}[3] = 'en:palm-kernel-oil-and-fat'
```

**Comment by stephanegigandet** [comment]
Can we close this pull request? We have another solution working for palm oil, and we should not have "XYZ" as a parent of "XYZ oil".

**Comment by aleene** [comment]
I guess we can close it.

**Pull request closed without merging**